src/d3a/d3a_core/sim_results/area_throughput_stats.py

```diff
@@ -36,10 +36,14 @@ def update_results(self, area_dict, core_stats, current_market_time_slot_str):
         area_throughput = core_stats.get(area_dict['uuid'], {}).get('area_throughput', {})
         imported_peak = round_floats_for_ui(area_throughput.get('imported_energy_kWh', 0.))
         exported_peak = round_floats_for_ui(area_throughput.get('exported_energy_kWh', 0.))
-        net_peak = round_floats_for_ui(area_throughput.get('net_energy_flow_kWh', 0.))
+        net_peak = imported_peak - exported_peak
+        import_peak_energy_net_kWh = net_peak if net_peak > 0 else 0.
+        export_peak_energy_net_kWh = abs(net_peak) if net_peak < 0 else 0.
         area_results = {
-            "import": {'peak_energy_kWh': imported_peak},
-            "export": {'peak_energy_kWh': exported_peak},
+            "import": {'peak_energy_trade_kWh': imported_peak,
+                       "peak_energy_net_kWh": import_peak_energy_net_kWh},
+            "export": {'peak_energy_trade_kWh': exported_peak,
+                       "peak_energy_net_kWh": export_peak_energy_net_kWh},
             "net_energy_flow": {'peak_energy_kWh': net_peak}
```

Comment:
Will the frontend use this too? If yes, please call `round_floats_for_ui` too, Thanks

Reply:
I will wait for Sarah to drop a comment in the ticket.